qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulationSymmetriseTabView.cpp

```diff
@@ -196,11 +218,26 @@ void InelasticDataManipulationSymmetriseTabView::xRangeLowChanged(double value)
  *
  * @param value New range selector value
  */
-void InelasticDataManipulationSymmetriseTabView::xRangeHighChanged(double value) {
+void InelasticDataManipulationSymmetriseTabView::notifyXrangeHighChanged(double value) {
   m_dblManager->setValue(m_properties["Ehigh"], value);
   m_dblManager->setMaximum(m_properties["Elow"], value);
 }
 
+void InelasticDataManipulationSymmetriseTabView::notifyReflectTypeChanged(QtProperty *prop, int value) {
+  if (prop->propertyName() == "ReflectType") {
+    QString workspaceName = m_uiForm.dsInput->getCurrentDataName();
+
+    if (validate()) {
+      m_presenter->handleReflectTypeChanged(value);
+
+      MatrixWorkspace_sptr sampleWS =
+          AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(workspaceName.toStdString());
+      auto const axisRange = getXRangeFromWorkspace(sampleWS);
+
+      resetEDefaults(value == 0, axisRange);
+    }
+  }
```

Comment:
At some point it would be good to move this logic to the presenter. Ideally we want the view to be as dumb as a pile of bricks - the reason being that we don't tend to write tests for the view due to it being a QObject, so to have test coverage for this logic it makes sense for it to be in the presenter

Reply:
Thanks. We could open some issues in the future to deal with this.